ai-platform-unified/notebooks/unofficial/sdk/metric-parameter-tracking-for-custom-jobs.ipynb

```diff
@@ -0,0 +1,1006 @@
+{
```

Comment:
<p>Please delete the training job and endpoint at cleanup. See template.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='67'/>

Reply:
<p>The one in template does not work for our case directly as the job, endpoint DIR are not saved. I followed the following example and used the delete function for the clean up. </p><p>https://github.com/GoogleCloudPlatform/ai-platform-samples/blob/master/ai-platform-unified/notebooks/official/custom/sdk-custom-image-classification-online.ipynb</p>

Reply:
Sure, thanks